app/controllers/main/routes.py

```diff
@@ -12,30 +13,31 @@ def home():
     print(f"{g.current_user.username}: {g.current_user.firstName} {g.current_user.lastName}")
     return render_template('main/home.html', title="Welcome to CELTS!")
 
-@main_bp.route('/volunteerIndicateInterest', methods = ['GET'])
-def volunteerIndicateInterest():
-    programs = Program.select()
-    interests = Interest.select().where(Interest.user == g.current_user)
-    interests_ids = [interest.program.id for interest in interests]
-    return render_template('volunteerIndicateInterest.html',
-                           title="Volunteer Interest",
-                           user = g.current_user,
-                           programs = programs,
-                           interests = interests,
-                           interests_ids = interests_ids)
-
-
 @main_bp.route('/profile/<username>', methods = ['GET'])
 def profilePage(username):
+    if not g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentStaff or g.current_user.username == username:
```

Comment:
the way this logic is written, a celts student staff member will receive a 403 Forbidden. And so would any student viewing their own profile